fix/dry-buckets.lua

```diff
@@ -1,28 +1,35 @@
 -- Removes water from buckets (for lye-making).
+-- Can also be used to remove water (muddy or not) that is blocking well operations.
 --[====[
 
 fix/dry-buckets
 ===============
 Removes water from all buckets in your fortress, allowing them
-to be used for making lye.  Skips buckets in buildings (eg a well),
-being carried, or currently used by a job.
+to be used for making lye. Skips buckets being carried, or currently used by a job.
 
 ]====]
```

Comment:
could you remove this entire doc comment and merge your doc changes into docs/fix/dry-buckets.rst?

when I copied all the documentation over, I didn't delete the original docstrings because at the time, we needed both systems to be functional.